Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroupBus.h

```diff
@@ -0,0 +1,72 @@
+
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <PrefabGroup/PrefabGroup.h>
+#include <SceneAPI/SceneCore/Containers/SceneGraph.h>
+#include <AzCore/Component/EntityId.h>
+
+namespace AZ::SceneAPI::DataTypes
+{
+    class ICustomPropertyData;
+}
+
+namespace AZ::SceneAPI::SceneData
+{
+    class PrefabGroup;
+}
+
+namespace AZ::SceneAPI::Containers
+{
+    class Scene;
+}
+
+namespace AZ::SceneAPI
+{
+    using PrefabGroup = AZ::SceneAPI::SceneData::PrefabGroup;
+    using Scene = AZ::SceneAPI::Containers::Scene;
+
+    //! Events that handle Prefab Group logic.
+    //! The behavior context will reflect this EBus so that it can be used in Python and C++ code.
+    class PrefabGroupEvents
```

Comment:
Should this be called PrefabGroupRequests for consistency?

Reply:
done